src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -196,140 +185,137 @@ async def _failed_handler(self, events: Sequence[EnsembleFailed]) -> None:
     def ensemble(self) -> Ensemble:
         return self._ensemble
 
-    @contextmanager
-    def store_client(self, websocket: ServerConnection) -> Generator[None, None, None]:
-        self._clients.add(websocket)
-        yield
-        self._clients.remove(websocket)
-
-    async def handle_client(self, websocket: ServerConnection) -> None:
-        with self.store_client(websocket):
+    async def handle_client(self, dealer: bytes, frame: bytes) -> None:
+        raw_msg = frame.decode("utf-8")
+        if raw_msg == "CONNECT":
```

Comment:
Should we have the `CONNECT` string as a constant in client.py, and call it CONNECT_MSG?

Reply:
Good suggestion! I thought about that and now it makes sense :+1:

Reply:
Done!